### PR TITLE
[4.0] Implementing RTL for admin menus

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Adding css in Atum template-rtl.scss

### Testing Instructions
Test by modifying `<rtl` tag in en-GB.xml. 
No need in this case to use Persian to test.

Use npm ci

### Reference
The menu as we get it for LTR
screen shot 2018-11-22 at 17 46 05

### Before PR in RTL
screen shot 2018-11-22 at 18 02 36

### After PR in RTL
screen shot 2018-11-22 at 17 48 06

@C-Lodder @Quy @wilsonge

### CODE REVIEW

File: administrator/templates/atum/scss/template-rtl.scss [hunk 1]
```diff
@@ -115,18 +115,38 @@ body {
...
-    float: right;
+  .has-arrow::after {
+    left: 1rem;
+    right:inherit;
+    -webkit-transform: rotate(135deg) translate(0,-50%);
+    transform: rotate(135deg) translate(0,-50%);
+  }
+
+  ul a::before {
+    right:18px;
```
Comment (Quy) [inline administrator/templates/atum/scss/template-rtl.scss]:
```suggestion
    right: 18px;
```

File: administrator/templates/atum/scss/template-rtl.scss [hunk 2]
```diff
@@ -115,18 +115,38 @@ body {
...
     right: auto;
     left: 15px;
     content: "\f0d9";
   }
 
-  li a span {
-    float: right;
+  .has-arrow::after {
+    left: 1rem;
+    right:inherit;
```
Comment (Quy) [inline administrator/templates/atum/scss/template-rtl.scss]:
```suggestion
    right: inherit;
```

File: administrator/templates/atum/scss/template-rtl.scss (hunk 1 above)
Comment (infograf768) [reply administrator/templates/atum/scss/template-rtl.scss]: done

File: administrator/templates/atum/scss/template-rtl.scss (hunk 2 above)
Comment (infograf768) [reply administrator/templates/atum/scss/template-rtl.scss]: done

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 1e7874dfcd74a9f27bf4a5f2087921faaac1e39b<br /><br />Tested with Persian language.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23143">issues.joomla.org/tracker/joomla-cms/23143</a>.</sub>

**Comment by infograf768** [comment]
Thanks for testing. need one more.

**Comment by Quy** [comment]
`Joomla! Update`, `Multilingual Associations` and `Post-Installation Messages` don't align with the other menus that have submenus.

**Comment by infograf768** [comment]
@Quy @richard67 
You are correct. Fine tuned these alignment.
Please test again 

We should now get:
screen shot 2018-11-26 at 08 52 34

**Comment by Quy** [comment]
For the submenus, move to the right slightly.

**Comment by infograf768** [comment]
@Quy 
moved 2px to the right

screen shot 2018-11-27 at 07 29 34

**Comment by infograf768** [comment]
Note to @Quy : we can play forever with slight modifications for these margins. The results are somehow a bit different when using Persian, Arabic or Hebrew as my experience shows it may depend on the font used.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ba1a189614bfad702f98733ee2eb8a9382753fee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23143">issues.joomla.org/tracker/joomla-cms/23143</a>.</sub>

File: administrator/templates/atum/scss/template-rtl.scss [hunk 3]
```diff
@@ -115,18 +115,38 @@ body {
...
     left: 15px;
     content: "\f0d9";
   }
 
-  li a span {
-    float: right;
+  .has-arrow::after {
+    left: 1rem;
+    right: inherit;
+    -webkit-transform: rotate(135deg) translate(0,-50%);
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/template-rtl.scss]: Don't use vendor prefixes. Autoprefix will take care of this

**Review by C-Lodder — CHANGES_REQUESTED** [review]

File: administrator/templates/atum/scss/template-rtl.scss (hunk 3 above)
Comment (infograf768) [reply administrator/templates/atum/scss/template-rtl.scss]: Sorry, I do not understand what you mean.

Comment (mbabker) [reply administrator/templates/atum/scss/template-rtl.scss]: You don't need to include vendor prefixed CSS rules in the SCSS files (i.e. the `-webkit-` stuff).  When the files are compiled to CSS, there is a package (autoprefixer) which handles adding these to the CSS as needed based on the supported browser rules.

Comment (infograf768) [reply administrator/templates/atum/scss/template-rtl.scss]: should it just be
`transform: rotate(135deg) translate(0,-50%);`
once ?

Comment (mbabker) [reply administrator/templates/atum/scss/template-rtl.scss]: Yes.

Comment (infograf768) [reply administrator/templates/atum/scss/template-rtl.scss]: done

**Pull request merged**